io/net/bonding.py.data/README.txt

```diff
@@ -31,5 +31,3 @@ Requirements:
 -----------------------
 1. install netifaces using pip
 command: pip install netifaces
-2. Generate sshkey for your test partner to run the test uninterrupted.(Have a passwordless ssh between the peers)
-3. Make sure IPs are set for interfaces to be used, via configuration file. ifup / ifdown should set the IPs back.
```

Comment:
why remove this info are we doing this in the test?

Reply:
both the requirements are already automated into script.. so they are not needed